src/Network/Send.pm

```diff
@@ -1221,4 +1221,42 @@ sub sendProduceMix {
 	debug "Sent Forge, Produce Item: $ID\n" , 2;
 }
 
+##
+# Request to merge item
+# 096E <size>.W { <index>.W }*
+# @author [Cydh]
+##
+sub sendMergeItemRequest {
+	my ($self, $num, $items) = @_;
+	my $len = ($num * 4) + 12;
+	$self->sendToServer($self->reconstruct({
+		switch => 'merge_item_request',
+		len => $len,
```

Comment:
Packet length should automatically be added (if packet lengths are configured properly) by `reconstruct`, no need to specify it:

https://github.com/OpenKore/openkore/blob/a668f7f8701974a995b51dfa9985631113bb8096/src/Network/PacketParser.pm#L195